keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/ManualJudgementNotifier.kt

```diff
@@ -31,32 +32,41 @@ class ManualJudgementNotifier(
   private val echoService: EchoService,
   private val repository: KeelRepository,
   private val scmInfo: ScmInfo,
+  private val springEnv: Environment,
   @Value("\${spinnaker.baseUrl}") private val spinnakerBaseUrl: String
 ) {
   companion object {
     const val MANUAL_JUDGEMENT_DOC_URL =
       "https://www.spinnaker.io/guides/user/managed-delivery/environment-constraints/#manual-judgement"
   }
 
+  private val isNewSlackEnabled: Boolean
+    get() = springEnv.getProperty("keel.notifications.slack", Boolean::class.java, true)
```

Comment:
Should we default to `false` if we can't retrieve the config?

Reply:
hmm i'm setting the FP to false in keel, and I plan to remove it pretty soon, not sure if it makes sense to change it now?

Reply:
Up to you... 🙂